frontend/src/metabase/dashboard/components/Dashboard/Dashboard.unit.spec.tsx

```diff
@@ -0,0 +1,188 @@
+import { renderWithProviders, waitForLoaderToBeRemoved } from "__support__/ui";
+import {
+  createMockDashboardCard,
+  createMockDashboard,
+} from "metabase-types/api/mocks";
+import type { Dashboard, Parameter } from "metabase-types/api";
+import { Dashboard as DashboardComponent } from "./Dashboard";
+
+type SetupOpts = {
+  dashboardId: number | null;
+  dashboard: Dashboard | null;
+  selectedTabId: number | null; // when there's no tabs, is null
+  parameters: Parameter[]; // when empty, is an empty array
```

Comment:
It feels like the code here is pretty self-explanatory and we probably don't need these comments